APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id
+        Delete Project:
+            Delete localhost:8080/API/v1/projects/id
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+    User Management:

```

Comment:
Add another API for `Product access` to set user access:
`User (default)`
`User Access Admin`
`Product Admin`